<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessMRI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>ProcessMRI

Simple tools to process MRI images.
First open an image with &quot;File/Open&quot;.
Second, use processing tools in &quot;Process&quot;:
- multi-exponential fit
- denoising with temporal phase correction
- denoising with non-local means</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionBruker_directory"/>
     <addaction name="actionNifti"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionExponential_fitting"/>
    <addaction name="actionDenoising_TPC"/>
    <addaction name="actionDenoising_NL_means"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_manual_FR"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcess"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBruker_directory">
   <property name="text">
    <string>Bruker directory</string>
   </property>
  </action>
  <action name="actionNifti">
   <property name="text">
    <string>Nifti</string>
   </property>
  </action>
  <action name="actionExponential_fitting">
   <property name="text">
    <string>Exponential fitting</string>
   </property>
  </action>
  <action name="actionDenoising_TPC">
   <property name="text">
    <string>Denoising TPC</string>
   </property>
  </action>
  <action name="actionDenoising_NL_means">
   <property name="text">
    <string>Denoising NL-means</string>
   </property>
  </action>
  <action name="actionUser_manual_FR">
   <property name="text">
    <string>User manual (FR)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
